.travis.yml

```diff
@@ -84,6 +84,12 @@ matrix:
     env: BEAKER_set="debian-9" BEAKER_PUPPET_COLLECTION=puppet6
     bundler_args:
     script: bundle exec rake beaker
+  - rvm: 2.5.3
```

Comment:
please add puppet 5

Reply:
With which rvm version? There is no puppetserver5 package for debian 10 available.

Reply:
It doesn't use the puppetserver, it uses the agent package.